src/Compilers/CSharp/Portable/Symbols/Source/GlobalExpressionVariable.cs

```diff
@@ -111,7 +113,9 @@ internal override TypeWithAnnotations GetFieldType(ConsList<FieldSymbol> fieldsB
             }
 
             diagnostics.Free();
+#nullable disable // The compiler can't tell that 'SetType' ensures '_lazyType' is not null. https://github.com/dotnet/roslyn/issues/39166
```

Comment:
Nothing the compiler will ever be able to do here. Please use an assertion instead of `#nullable disable`